src/coreclr/gc/gc.cpp

```diff
@@ -2201,8 +2214,34 @@ size_t      gc_heap::committed_by_oh[total_oh_count] = {0, 0, 0, 0};
 
 size_t      gc_heap::current_total_committed_bookkeeping = 0;
 
+#ifdef FEATURE_EVENT_TRACE
```

Comment:
is `FEATURE_EVENT_TRACE` not enabled by default?

Reply:
it's enabled by default but it is a "feature", as in, it may not be defined.

Reply:
Yes, and there are definitely folks that undef `FEATURE_EVENT_TRACE`.